src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/GrindStone.java

```diff
@@ -91,6 +91,9 @@ protected void registerDefaultRecipes(@Nonnull List<ItemStack> recipes) {
 
         recipes.add(SlimefunItems.ENDER_LUMP_3);
         recipes.add(new SlimefunItemStack(SlimefunItems.ENDER_LUMP_2, 4));
+
+        recipes.add(new ItemStack(Material.DIAMOND));
+        recipes.add(new ItemStack(Material.COAL, 24));
```

Comment:
Why 24 though?

Reply:
I felt like 16 was a bit too few and 32 a bit too much
What would you suggest?

Reply:
I did the maths, and it seems like 1 diamond (synthesized) corresponds to 8 Compressed Carbon which each correspond to 4 Carbon which each correspond to 8 coal.
That means 1 diamond is roughly worth 4 stacks of coal.
Making this a 1:1 mapping seems ridicilous, I admit.
But I think 32 is a bit more on the fair side here while still yielding a big loss of coal compared to the other way around.

Another idea on how to spice this up is to introduce a middle item.
How about you grind diamonds into e.g. 8 diamond-coal-hybrid item and then these can each be ground into 4 coal.
Or similar.

Reply:
All right

Reply:
I was looking for a good name and then I realized we already have Carbons, we can just grind a diamond into 4 Carbon, and then just add a way to convert 1 carbon back into 8 coal.
That seem good?